include/EVT/dev/BitmapFonts.hpp

```diff
@@ -0,0 +1,252 @@
+/**
+* BitMapFont is a class that holds a representation of a bitmap font. The class can hold multiple fonts, however
+ * currently it is only holding a single 4x6 font.
```

Comment:
This comment should be  immediately before the class declaration, and the header guards should be at the top. This one has to be near the class instead of at the top of the file, so the documentation generator can find it.